dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/datavalue/DataValidator.java

```diff
@@ -264,6 +270,33 @@ public DataSet getAndValidateOptionalDataSet( final String uid, final DataElemen
         return dataSet;
     }
 
+    /**
+     * Validates and retrieves a data value.
+     *
+     * @param dataValueDto the {@link DataValueRequest}.
+     * @return a data value.
+     * @throws IllegalQueryException if the validation fails.
+     */
+    public DataValue getAndValidateDataValue( DataValueRequest dataValueDto )
```

Comment:
`dataValueDto` maybe should also be renamed (like the new class name)

Reply:
Fixed.